extensions/OpenAI/OpenAI/OpenAI.csproj

```diff
@@ -15,7 +15,9 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.SemanticKernel.Connectors.OpenAI" />
-        <PackageReference Include="Microsoft.ML.Tokenizers" /> <!-- prerelease dependency (NU5104) -->
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.Cl100kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.O200kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.P50kBase" />
```

Comment:
```suggestion
```

Actually these can be removed, the dependency was not used. All tokenizers are imported via Tiktoken.csproj